### DISCUSSION TITLE
Post CSS plugin issue in nuxt js 2.*

### DISCUSSION THREAD

**Discussion by Dipakbisen** [nuxt/nuxt#10604] [General]
![image](https://user-images.githubusercontent.com/51753012/181166478-161f1d4f-27fb-4812-8679-57a5650c96cf.png)

nuxt.config.js

```
build: {
    postcss: {
      plugins: {
        'postcss-url': false,
        lost: true,
        // 'postcss-nested': {},
        // 'postcss-responsive-type': {},
        // 'postcss-hexrgba': {}
      },
      preset: {
        autoprefixer: {
          grid: true,
        },
      },
    },
  },
```

**Comment by ghost** [comment]
> ![imagen](https://user-images.githubusercontent.com/51753012/181166478-161f1d4f-27fb-4812-8679-57a5650c96cf.png)
> 
> nuxt.config.js
> 
> ```
> build: {
>     postcss: {
>       plugins: {
>         'postcss-url': false,
>         lost: true,
>         // 'postcss-nested': {},
>         // 'postcss-responsive-type': {},
>         // 'postcss-hexrgba': {}
>       },
>       preset: {
>         autoprefixer: {
>           grid: true,
>         },
>       },
>     },
>   },
> ```

**Reply by Dipakbisen** [reply]
?
